public/script/askAwayButton.js

```diff
@@ -0,0 +1,3 @@
+document.getElementById("createButton").onclick = function () {
+location.href = "http://localhost:3000/poll/create";
```

Comment:
do `location.href = "/poll/create"`. that way it wont break on a different host and/or port